CHANGELOG.md

```diff
@@ -0,0 +1,260 @@
+# Change Log
+
+All notable changes to this project will be documented in this file. This
+project adheres to [Semantic Versioning](http://semver.org).
+
+## [Unreleased]
+
+### Added
+
+- Added core grid functionality in `grid-column`
+- Added improved documentation and releasing information
+- Added `grid-container` for floated grid which contains a simple clearfix
+- Added `bundle exec` for rake release task
+- Added `grid-push` functionality
+- Added `neat-setting-retrieve()` function
+- Added `grid-calc` to abstract the calculation of column widths
+
+### Changed
+
+- Change to https for RubyGems gem source
+- Update gulpfile to watch contrib styles and not just core project files
+- Use `core/` as the primary Neat source directory
+
+### Removed
+
+- Removed all previous Neat Sass
```

Comment:
It might be wise to break this out into the exact user features that have been removed, so that they can use this as a reference. Meaning, “`span-columns` has been removed”, etc.